### PR TITLE
Added PHPixie DI

### PR DESCRIPTION
**Pull request by dracony**
Added PHPixie DI to the showdown

### CODE REVIEW

File: src/Container/PHPixieDi/Container/PrototypeContainer.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        
+        $this->callback('c1', function() use($class) {
+            return new $class;
+        });
+        
+        for ($i = 2; $i <= 100; $i++) {
+            $class = $prefix . $i;
+            $dependency = 'c' . ($i-1) ;
+            $this->callback('c' . $i, function() use($class, $dependency) {
+                return new $class($this->getValue($dependency, true, []));
```
Comment (kocsismate) [inline src/Container/PHPixieDi/Container/PrototypeContainer.php]: You could also generate the configuration code e.g. on 3v4l in order to save some bootstrap time. And as far as I know `new $class` invocations will be slower than instantiating e.g.`Class2` directly. But I pleasingly accept your PR if you think that this is config is good for you.

**Comment by dracony** [comment]
Pregenerated config is a good idea actually. For some reason I thought it
would count as cheating :)

In that case I'll push a pregenerated version tomorrow (already sleeping at
the moment).

Anyway thanks for the benchmarking suite, it's cool.

**Comment by kocsismate** [comment]
Haha :) Here is some shortcut for you: https://github.com/kocsismate/php-di-container-benchmarks/blob/master/src/Container/ZendServiceManager/AbstractZendServiceManagerTest.php#L120

So I am looking forward to your new commit (there will be a small conflict in the composer.json by then)!

**Comment by dracony** [comment]
Done, and rebased to fix the conflicts =)

**Pull request merged**